<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flash_card</class>
 <widget class="QMainWindow" name="Flash_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new_qrc/SoftNote.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="card_list"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Cards</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_date_time">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="back_button">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="usesScrollButtons">
           <bool>false</bool>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Flash cards</string>
           </attribute>
           <attribute name="toolTip">
            <string>create some cards</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="2">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPushButton" name="add_text">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="update_text">
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_text">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clear_text">
                <property name="text">
                 <string>clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="tquestion_label">
                <property name="text">
                 <string>Question:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="tquestion_text"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="tanswer_label">
                <property name="text">
                 <string>Answer:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTextEdit" name="tanswer_text"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_11"/>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Study</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string/>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ink Free</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);
font: 900 italic 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;
font: 10pt &quot;Ink Free&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;TOPICS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="new_topic"/>
        </item>
        <item>
         <widget class="QPushButton" name="new_topic_button">
          <property name="text">
           <string>Add New Topic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>400</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>300</width>
             <height>400</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actiontext_color"/>
    <addaction name="actionbeckground_color"/>
    <addaction name="actionbackground"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout_us_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuAbout_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionnew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionload">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionbeckground_color">
   <property name="text">
    <string>Background 1</string>
   </property>
  </action>
  <action name="actionbackground">
   <property name="text">
    <string>Background 2</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionabout_us_2">
   <property name="text">
    <string>about us</string>
   </property>
  </action>
  <action name="actiontext_color">
   <property name="text">
    <string>Text color</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
